BusinessLogicLayer/seatsBLL.cs

```diff
@@ -20,14 +20,24 @@ public IEnumerable<Seats> GetAllSeats()
             return _seat.GetAllSeats();
         }
 
+        public void Insert(Seats seat)
+        {
+            _seat.Insert(seat);
+        }
+
+        public void Update(Seats seat)
+        {
+            _seat.Update(seat);
+        }
+
         public IEnumerable<Seats> GetSeatsByBus(int id, string status)
         {
```

Comment:
The code bellow is doing the same thing, but only status is changed. therefore you don't need to use if ()
i.e.: 
List<Seats> seats = _seat.GetAllSeats().Where(x => x.BusId == id && x.Status == status).ToList();
return seats;